### PR TITLE
Network march

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
## Description

March release for BUILD

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -191,6 +191,10 @@
         - New-AzApplicationGatewayRewriteRuleCondition
     - Cmdlets updated with optional parameter - RuleSequence and Condition
         - New-AzApplicationGatewayRewriteRule
+* Added support for updating tags in Set operations for VirtualNetworkGateways and VirtualNetworkGatewayConnections
```
Comment (markcowl) [inline ChangeLog.md]: There shouldn't be any changes here

Comment (MikhailTryakhov) [reply ChangeLog.md]: This PR contains only alredy approved PRs by PowerShell team... @MiYanni for visibility.
Not sure why can't we add changes to changeLog?

**Comment by markcowl** [comment]
@MikhailTryakhov Just requesting revert to the root level changelog, otherwise LGTM

**Comment by MikhailTryakhov** [comment]
@markcowl I'll have to make separated PR from my local branch in separated PR

**Comment by MikhailTryakhov** [comment]
closed due to https://github.com/Azure/azure-powershell/pull/9092

**Pull request closed without merging**